### ISSUE TITLE
website screenshots' alt text should be more descriptive (ideally news headline or publication)

### ISSUE THREAD

**Issue by matthewfdaniels** [the-pudding/sleep-training#71]
<img width="884" alt="image" src="https://github.com/user-attachments/assets/f6d8adc5-8740-46d1-8bed-eb085827c513">

**Comment by tomvaillant** [comment]
Not sure if this was in or out of the staging you tested apologies, I added the altTag in the conditional logic selecting the image folders, which I thought solved this - but maybe you're asking for more detail so double-checking.

```const getImageUrls = (album) => {
    if (album === 'articles') {
      altTag = "Image carrousel of article headlines on sleep training.";
      return [
        { url: 'assets/images/articles/article_1.jpg' },
        { url: 'assets/images/articles/article_2.jpg' },
        { url: 'assets/images/articles/article_3.jpg' },
        { url: 'assets/images/articles/article_4.jpg' },
        { url: 'assets/images/articles/article_5.jpg' },
        { url: 'assets/images/articles/article_6.jpg' },
      ];
    } else if (album === 'social') {
      altTag = "Image carrousel of article headlines on the sleep training debate.";
      return [
        { url: 'assets/images/social/social_1.png' },
        { url: 'assets/images/social/social_2.png' }
      ];
    } else if (album === 'posts') {
      altTag = "Social media post images on anti-sleep training narratives.";
      return [
        { url: 'assets/images/posts/post_1.jpg' },
        { url: 'assets/images/posts/post_2.jpg' },
        { url: 'assets/images/posts/post_3.jpg' },
        { url: 'assets/images/posts/post_4.jpg' },
      ];
    } else if (album === 'debunk') {
      return [
        { url: 'assets/images/debunk/narvaez.jpg' },
        { url: 'assets/images/debunk/sears.jpg' },
      ];
    } 
    return [];
  };````

**Comment by matthewfdaniels** [comment]
the main goal for this exercise is to provide a reasonable experience for the visually impaired. e.g., "All vital information is available non-visually."

there's not a ton of images here, so it's reasonable to write an alt text description for each image that includes the content of the image. for the news articles, use the headline and publication (if it's in the image)...so "Screenshot of article from [publication] titled [title]". for the social media images, describe the image and/or include the text.

**Comment by tomvaillant** [comment]
Done :)
